library/src/main/java/io/ona/kujaku/activities/MapActivity.java

```diff
@@ -371,25 +419,30 @@ public void showAlertDialog(int title, int message,
     @Override
     protected void onResume() {
         super.onResume();
-        if (mapView != null) mapView.onResume();
+        List<Point> droppedPoints = null;
```

Comment:
Sorry but it's not clear what is meant here. Do you mean removing that line would cause that.

This code line may be outdated. Please check if this issue was fixed in subsequent pushes.